src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/stringliteralequality/InputStringLiteralEqualityCheck.java

```diff
@@ -0,0 +1,39 @@
+/*
+StringLiteralEquality
+
+
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.coding.stringliteralequality;
+
+// ok
```

Comment:
Remove this extra comment, as you have violation

Reply:
Done